src/tests/intg/test_enumeration.py

```diff
@@ -421,7 +421,7 @@ def _test_add_remove_user(ldap_conn, blank_rfc2307):
 def _test_add_remove_group_rfc2307(ldap_conn, blank_rfc2307):
     """Test RFC2307 group addition and removal are reflected by SSSD"""
     e = ldap_ent.group(ldap_conn.ds_inst.base_dn, "group", 2001)
-    time.sleep(INTERACTIVE_TIMEOUT/2)
+    time.sleep(INTERACTIVE_TIMEOUT / 2)
```

Comment:
Could you explain why it is not a problem with python-pep8-1.6.2-9?

Reply:
It depends whether E226 is to be ignored by default or not.

It seems that in all distros we use it's not a problem, so I'll drop this part (not only here, but in other patches with similar problems).